pkg/model/volumes.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2020 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/okteto/okteto/pkg/errors"
+)
+
+// HasLocalVolumes returns true if the manifest has local volumes
+func (dev *Dev) HasLocalVolumes() bool {
+	for _, v := range dev.Volumes {
+		if v.LocalPath != "" {
+			return true
+		}
+	}
+	return false
+}
+
+// HasRemoteVolumes returns true if the manifest has remote volumes
+func (dev *Dev) HasRemoteVolumes() bool {
+	for _, v := range dev.Volumes {
+		if v.LocalPath == "" {
+			return true
+		}
+	}
+	return false
+}
+
+// IsSyncFolder returns if path must be synched
+func (dev *Dev) IsSyncFolder(path string) (bool, error) {
```

Comment:
for which scenario will this return false?

Reply:
data volumes. or if you have something like this:
```
volumes:
   - .:/app
   - config:/config
```
the second one is not a sync folder, it is a subpath of the first one and it is not managed by synchthing, but just using a k8s volume mount with the proper subpath